doc/tutorial/essentials_quiz

```diff
@@ -0,0 +1,555 @@
+                            </li>
+                            <li>
+                                <p class="question">What command would you use to retrieve some code from a repository?</p>
+                                <div id="2" class="questions">
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm checkout</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm retrieve</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm get</label><br/></li>
+                                        <li><input type="radio" value="D" /><label for="D">fcm branch</label><br/></li>
+                                    </ol>
+                                    <p class="answers A"><em>A</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">What command would not checkout code from the repository?</p>
+                                <div id="3" class="questions">
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm co</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm ct</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm checkout</label><br/></li>
+                                    </ol>
+                                    <p class="answers D"><em>B</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">Which commands would commit code to the repository?</p>
+                                <div id="4" class="questions">
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm commit|ct|ci</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm commit|ct</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm commit|ci</label><br/></li>
+                                        <li><input type="radio" value="D" /><label for="D">fcm ct|ci</label><br/></li>
+                                    </ol>
+                                    <p class="answers C"><em>C</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">What is the best command for finding out in what aspects your working copy is different from the repository, i.e. what has been modified etc?</p>
```

Comment:
I think you want to say what files have changes rather than in what aspects your working copy is different. I was expecting fcm diff to be an option based on that question.